src/actions/activator.test.ts

```diff
@@ -123,75 +123,75 @@ describe('GetProvisioningCertObj', () => {
-    expect(clientObj.hostname).toBe(clientObj.ClientData.payload.hostname)
-  })
-  test('should populate fwNonce and modes after IPS_HostBasedSetupService call', async () => {
-    const message = { payload: { IPS_HostBasedSetupService: { response: { ConfigurationNonce: 'abcd1234=', AllowedControlModes: [2, 1] } } } }
-    const clientObj = clientManager.getClientObject(clientId)
-    await activator.processWSManJsonResponse(message, clientId)
-    expect(clientObj.ClientData.payload.fwNonce).toBeDefined()
-    expect(clientObj.ClientData.payload.modes).toEqual([2, 1])
-  })
-  test('should throw an error if adding a certificate to AMT fails', async () => {
-    const message = { payload: { Header: { Method: 'AddNextCertInChain' }, Body: { ReturnValue: 1 } } }
-    const clientObj = clientManager.getClientObject(clientId)
-    try {
-      await activator.processWSManJsonResponse(message, clientId)
-    } catch (error) {
-      expect(error.message).toEqual(`Device ${clientObj.uuid} activation failed. Error while adding the certificates to AMT.`)
-    }
-  })
-  test('logger reports certificate was added to device', async () => {
-  })
-  test('should throw an error if ACM activation fails', async () => {
-    const message = { payload: { Header: { Method: 'AdminSetup' }, Body: { ReturnValue: 1 } } }
-    const clientObj = clientManager.getClientObject(clientId)
-    try {
-      await activator.processWSManJsonResponse(message, clientId)
-    } catch (error) {
-      expect(error.message).toEqual(`Device ${clientObj.uuid} activation failed. Error while activating the AMT device in admin mode.`)
-    }
-  })
-  test('should populate ciraconfig.status and activationStatus if ACM activation successful', async () => {
-  })
-  test('should throw an error if CCM activation fails', async () => {
-    const message = { payload: { Header: { Method: 'Setup' }, Body: { ReturnValue: 1 } } }
-    const clientObj = clientManager.getClientObject(clientId)
-    try {
-      await activator.processWSManJsonResponse(message, clientId)
-    } catch (error) {
-      expect(error.message).toEqual(`Device ${clientObj.uuid} activation failed. Error while activating the AMT device in client mode.`)
-    }
-  })
-  test('should populate ciraconfig.status and activationStatus if CCM activation successful', async () => {
-  })
-  test('should thow an error if receives an invalid response', async () => {
-    const message = { payload: { Header: { Method: 'badrequest' }, Body: { ReturnValue: 0 } } }
-    const clientObj = clientManager.getClientObject(clientId)
-    try {
-      await activator.processWSManJsonResponse(message, clientId)
-    } catch (error) {
-      expect(error.message).toEqual(`Device ${clientObj.uuid} sent an invalid response.`)
-    }
-  })
-  test('logger reports that MEBx password was updated', async () => {
+  // test('should throw an error if the digest realm is invalid', async () => {
```

Comment:
instead of commenting all these out, can we just update them to work? all thats really needed is an HTTP handler right?

Reply:
done